test/DebugInfo/debug_compilation_dir.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swiftc_driver -g \
+// RUN:   -wmo -c -debug-compilation-dir /path/to \
+// RUN:   %s -o - -emit-ir | %FileCheck --check-prefix=CHECK-ABS %s
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t
+// RUN: cd %t
+// RUN: cp %s .
+// RUN: %target-swiftc_driver -g \
+// RUN:   -wmo -c -debug-compilation-dir /path/to \
+// RUN:   debug_compilation_dir.swift -o - -emit-ir | %FileCheck --check-prefix=CHECK-REL %s
+
+func foo() {}
+
+// CHECK-ABS: !DIFile(filename: "{{.*}}/debug_compilation_dir.swift", directory: "/path/to")
+// CHECK-REL: !DIFile(filename: "debug_compilation_dir.swift", directory: "/path/to")
```

Comment:
maybe a test for `.` since that's your intended use?

Reply:
Thanks for the advice. Should add that one as well.

Reply:
this test will probably not work on Windows. You can check other tests that uses path separators for the correct regex to use.